instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/util.py

```diff
@@ -20,15 +20,14 @@
 def _extract_conn_attributes(conn_kwargs):
     """ Transform redis conn info into dict """
     attributes = {
-        "db.type": "redis",
```

Comment:
Span name for redis needs to be fixed too.

Reply:
This is span attribute. The span name for redis is `redis.command`.

Reply:
Yes, I only highlighted this because Github didn't let point to the above code.

Simiarly to pymemcache for redis, is there not a concept of "statement" or "command"? The string "redis.command" doesn't really tell me much in terms of what logical operation redis is doing.

Reply:
> The string "redis.command" doesn't really tell me much in terms of what logical operation redis is doing.

Agree.